capa/features/extractors/vmray/extractor.py

```diff
@@ -55,12 +55,16 @@ def extract_file_features(self) -> Iterator[Tuple[Feature, Address]]:
     def extract_global_features(self) -> Iterator[Tuple[Feature, Address]]:
         yield from self.global_features
 
+    def get_import_names(self) -> Iterator[Tuple[Feature, Address]]:
+        for filename_id, filename_data in self.analysis.filenames.items():
```

Comment:
filenames are imports?

Reply:
This information should be extracted from https://github.com/r-sm2024/vmray/blob/0676b2480da7f6a09047e00d64d093805de2cb3a/capa/features/extractors/vmray/models.py#L76-L81

Reply:
> filenames are imports?
I was confused and thought "filenames" from the flog.xml were the names of the imports. Sorry about that. I'm changing the method to extract from PEFileImports.